src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -58,6 +58,8 @@ def read_patchinfo
     @binary = []
     @packager = @file.value(:packager)
     @version = @file.value(:version)
+    @package_list = []
+    @file.each(:package) { |package| @package_list << package.text }
```

Comment:
`.map`?

Reply:
@hennevogel 

> `.map` ?

`.map` method is not defined for `@file`, that is an ActiveXML::Node object.